audformat/core/utils.py

```diff
@@ -285,8 +285,49 @@ def duration(
     return (ends - starts).sum()
 
 
+def expand_file_path(
+        index: pd.Index,
+        root: str,
+) -> pd.Index:
+    r"""Expand index with relative file path.
+
+    Args:
+        index: index with relative file path conform to
+            :ref:`table specifications <data-tables:Tables>`
+        root: directory expand the file path with
+
+    Returns:
+        index with absolute file path
+
+    Raises:
+        ValueError: if index is not conform to
+            :ref:`table specifications <data-tables:Tables>`
+
+    Example:
+        >>> index = filewise_index(['f1', 'f2'])
+        >>> index
+        Index(['f1', 'f2'], dtype='object', name='file')
+        >>> expand_file_path(index, '/some/where')  # doctest: +SKIP
```

Comment:
There should be no reason to skip this test here as it should be independent of the operating system and structure of underlying file system.

Reply:
Ah, yes, you are right due to `os.path.sep` it might indeed end up as:
```
/some/where\f1
```
That's in general a problem we have, publishing a database on Windows and using it on Linux will fail.

Reply:
At least I expect that it will not work ;)